.gitlab/tests/contrib.yml

```diff
@@ -194,3 +194,73 @@ wsgi:
   extends: .test_base_riot_snapshot
   variables:
     SUITE_NAME: "wsgi"
+
+langchain:
+  extends: .test_base_riot_snapshot
+  parallel: 6
+  variables:
+    SUITE_NAME: "langchain"
+
+anthropic:
+  extends: .test_base_riot_snapshot
+  parallel: 3
+  variables:
+    SUITE_NAME: "anthropic"
+
+loguru:
+  extends: .test_base_riot_snapshot
+  variables:
+    SUITE_NAME: "loguru"
+
+logbook:
+  extends: .test_base_riot_snapshot
+  variables:
+    SUITE_NAME: "logbook"
+
+asynctest:
+  extends: .test_base_riot
+  variables:
+    SUITE_NAME: 'asynctest$'
+
+pytest_v2:
+  extends: .test_base_riot_snapshot
+  parallel: 10
+  variables:
+    SUITE_NAME: 'pytest_plugin_v2'
+
+django_hosts:
+  extends: .test_base_riot_snapshot
+  variables:
+    SUITE_NAME: 'django_hosts$'
+
+test_logging:
+  extends: .test_base_riot_snapshot
+  variables:
+    SUITE_NAME: 'test_logging'
+
+stdlib:
+  extends: .test_base_riot_snapshot
+  variables:
+    SUITE_NAME: 'asyncio$|sqlite3$|futures$|dbapi$|dbapi_async$'
+
+llmobs:
+  extends: .test_base_riot
+  variables:
+    SUITE_NAME: "llmobs"
```

Comment:
nit: Should we move this into its own file too (so we can assign LLMObs as owners?)

Reply:
good call